app/assets/stylesheets/course/announcements.css.scss

```diff
@@ -0,0 +1,30 @@
+// Place all the styles related to the course/announcements controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+@import "variables";
+
+@mixin future($bg-color: $gray-lighter) {
```

Comment:
yes, but place this in a separate file. the future style will be used by other pages as well.